orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/util/PackageInfo.java

```diff
@@ -52,7 +52,15 @@ public PackageInfo(Stage stage, String packageType, String versionDelimiter, boo
     this.mapper = mapper;
     this.buildDetailExtractor = new BuildDetailExtractor();
 
-    packageFilePattern = Pattern.compile(format("%s.*\\.%s", stage.getContext().get("package"), packageType));
+    // can be a space separated set of packages
+    if (stage.getContext().containsKey("package")) {
+      String packages = stage.getContext().get("package").toString();
```

Comment:
`package` can actually be a space-separated set of packages ... in such cases a regex of:
`package1 package2.*\.deb` does not make a lot of sense.

This worked in the old world because the Jenkins stage emitted `buildInfo` to the global context which happened to short circuit the call to `artifactMatch()`.